lib/tennis_game.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# TennisGame: used to manage the scoring in a tennis game
+class TennisGame
+  TENNIS_POINTS = %w(Love Fifteen Thirty Fourty).freeze
```

Comment:
Minor comment but I'd be also tempted to make the above numbers as a String (e.g. `0`, `15`)